mne/viz/raw.py

```diff
@@ -822,8 +830,10 @@ def _plot_raw_traces(params, color, bad_color, event_lines=None,
             params['ax'].text((start + end) / 2., ylim[0], dscr, ha='center')
 
     # finalize plot
-    params['ax'].set_xlim(params['times'][0],
-                          params['times'][0] + params['duration'], False)
+    first_time = params['raw'].first_samp / float(sfreq)
```

Comment:
given then number of times `raw.first_samp / raw.info['sfreq']` shows up in this PR alone (at least 6), we might want to add a convenience private parameter as `raw._first_time`